Services/User/classes/class.ilRoleAssignmentTableGUI.php

```diff
@@ -193,26 +214,14 @@ public function parse(int $usr_id): void
             ];
             $this->ctrl->setParameterByClass("ilobjrolegui", "ref_id", $rolf[0]);
             $this->ctrl->setParameterByClass("ilobjrolegui", "obj_id", $role["obj_id"]);
```

Comment:
I think we cannot set these two parameters here and have to move them to `ilRoleAssignmentTableGUI::fillRow()`, too.

Reply:
Good catch @kergomard , I'll add the `obj_id` to the array.